samcli/commands/local/lib/local_lambda.py

```diff
@@ -256,7 +258,12 @@ def _make_env_vars(self, function: Function) -> EnvironmentVariables:
         else:
             # Standard format
             LOG.debug("Environment variables overrides data is standard format")
-            overrides = self.env_vars_values.get(name, None)
+            # Precedence: function_id -> logical_id -> full_path, customer can use any of them
```

Comment:
Why is this the right order? Shouldn't the previous one (logical_id) be the first to be backwards compatible?

Reply:
Thanks for pointing it out! I have fixed this order.